packages/expandable-search-bar/index.js

```diff
@@ -0,0 +1,5 @@
+import ExpandableSearchBar from './lib/expandable-search-bar.js';
```

Comment:
this is identical to the `index.d.ts` - i don't think it was identical in the `radio-player` PR - should they be the same like this file?

Reply:
It's _nearly_ identical ;) The `.d.ts` file imports the `.d` "declaration" files and this one imports the actual javascript "implementation" files.. ie: 

```
import ExpandableSearchBar from './lib/expandable-search-bar.d';  <-- /.d
vs
import ExpandableSearchBar from './lib/expandable-search-bar.js';  <-- /.js
```